syntax = "proto3";

package benchmark;

service EchoService {
	rpc Echo (EchoRequest) returns (EchoResponse) {}
}

message EchoRequest {
	string name = 1;
	int32 id = 2;
}

message EchoResponse {
	string response = 1;
}
